docs/source/model_design/general_reference_material/bliss/race_ethnicity/index.rst

```diff
@@ -402,17 +402,104 @@ feasible based on the data present, it is also a reasonable approach.
 3.0 Results
 +++++++++++
 
+3.0 Results
++++++++++++
+
+Communication of results
+------------------------
+
+In this section we’ll discuss how health metric researchers can talk about health research related to 
+race and ethnicity without stigmatizing, othering, or otherwise causing harm to historically marginalized 
+racial and ethnic groups. It is important to be aware of how we talk about race and racism, 
+particularly in the field of health metrics sciences, our goal is often to uncover and scrutinize the 
```

Comment:
```suggestion
particularly in the field of health metrics sciences, as our goal is often to uncover and scrutinize the 
```

This sounds like a run-on sentence, but I think you can fix it by adding 'as', unless that changes your intended meaning.